frontend/src/App.css

```diff
@@ -81,7 +81,7 @@ html {
   }
 }
 
-@media (max-width: 790px) {
+@media (max-width:950px) {
```

Comment:
Changed this to 950px to align with the navbar 950px change as well. This will change the entire horizontal view for Findr to a cleaner view.